content/docs/add-react-to-a-website.md

```diff
@@ -80,6 +80,7 @@ After **[the starter code](https://gist.github.com/gaearon/0b180827c190fe4fd98b4
 ```js{3,4}
 // ... the starter code you pasted ...
 
+const e = React.createElement;
```

Comment:
Notice the `// ... the starter code you pasted ...` a few lines above which refers to https://gist.githubusercontent.com/gaearon/0b180827c190fe4fd98b4c7f570ea4a8/raw/b9157ce933c79a4559d2aa9ff3372668cce48de7/LikeButton.js which already includes the code you added.

Does this clear things up?

Reply:
Oh, ok, thanks. I have to wonder why you decided to hide such crucial part of code over a link, but I guess it's my bad for not reading all the preceding text.